gradle.md

```diff
@@ -47,52 +50,79 @@ Technically speaking, `clean` should not be required (every time it is required
 
 ## Static checks
 
+### Release Audit Tool
+
       # Run RAT
       gw rat
+      
+### Code Formatting
+      
       # Run checkstyle for main (non-test) code
       gw checkstyleMain
+      
       # Run checkstyle for test code
       gw checkstyleTest
-
-## Compiling code
+      
+      # Run checkstlye for all
+      gw checkstyleAll
+      
+      # Run Spotless checks
+      gw spotlessCheck
+      
+      # Fix any issues found by Spotless
+      gw spotlessApply
+      
+      # Run spotlessApply and checkstyleAll
+      gw style
```

Comment:
Could you please move `gw style` and `gw checkstyle` commands to the first?
The rest commands are lowlevel, and I don't expect people would need them.